src/PHPSemVerChecker/Analyzer/ClassMethodAnalyzer.php

```diff
@@ -109,6 +112,29 @@ public function analyze(Stmt $contextBefore, Stmt $contextAfter)
 					);
 				}
 
+				if ($methodBefore->returnType !== $methodAfter->returnType) {
+					$class = ClassMethodCaseChanged::class;
```

Comment:
```suggestion
					$class = null
```
I don't think that by default it means a `ClassMethodCaseChanged` :smile:

By default, let's assume that even if the two types don't match, we might not add any detected change. Only if we recognize a specific change of type should we add it.